showcase/src/Sizes.elm

```diff
@@ -4,33 +4,42 @@ import Element
 import Element.Background as Background
 import Element.Font as Font
 import Msg as Msg
+import PluginOptions exposing (defaultWithMenu)
 import UI.Button as Button
 import UI.Icon as Icon
 import UI.Internal.Palette as Palette
 import UI.Size as Size
 import UI.TextField as TextField
 import UIExplorer exposing (storiesOf)
-import Utils exposing (iconsSvgSprite)
+import Utils exposing (iconsSvgSprite, prettifyElmCode)
 
 
 stories cfg =
     storiesOf
         "Sizes"
         [ ( "Large"
           , \_ -> sizeView cfg Size.large
-          , { note = notes "large" }
+          , { defaultWithMenu
+                | note = notes "large"
+            }
           )
         , ( "Medium"
           , \_ -> sizeView cfg Size.medium
-          , { note = notes "medium" }
+          , { defaultWithMenu
+                | note = notes "medium"
+            }
           )
         , ( "Small"
           , \_ -> sizeView cfg Size.small
-          , { note = notes "small" }
+          , { defaultWithMenu
+                | note = notes "small"
+            }
           )
         , ( "ExtraSmall"
           , \_ -> sizeView cfg Size.extraSmall
-          , { note = notes "extraSmall" }
+          , { defaultWithMenu
+                | note = notes "extraSmall"
```

Comment:
Actually, these notes are code samples/tutorial.

Reply:
I think this is more a "pseudocode", is not a proper code as it has the comments there. So maybe it makes sense keep it as notes and/or do a complete snippet to put in the code tab.

WDYT?

Reply:
If it has code or code with instructions, it is code.
If it has usage notes, warnings, design decisions, it is a note.